src/main/java/ch/njol/skript/expressions/ExprArmorChangeItem.java

```diff
@@ -0,0 +1,64 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.EventValueExpression;
+import ch.njol.skript.lang.EventRestrictedSyntax;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import com.destroystokyo.paper.event.player.PlayerArmorChangeEvent;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Armor Change Item")
+@Description("Get the unequipped or equipped armor item from a 'armor change' event.")
+@Examples("""
+	on armor change
+		broadcast the old armor item
+	"""
+)
```

Comment:
```suggestion
	""")
```
had a typo in the previous request, sorry